src/components/WebexInMeeting/WebexInMeeting.stories.js

```diff
@@ -7,7 +7,7 @@ export default {
   component: WebexInMeeting,
 };
 
-const Template = (args) => {
+const Template = (...args) => {
```

Comment:
Why are these changes needed in the stories?

Reply:
These were giving errors